test/e2e/scheduling/nvidia-gpus.go

```diff
@@ -186,7 +186,7 @@ func dsFromManifest(url string) *extensions.DaemonSet {
 	return &controller
 }
 
-var _ = framework.KubeDescribe("[Feature:GPU]", func() {
+var _ = framework.KubeDescribe("[sig-scheduling] [Feature:GPU]", func() {
```

Comment:
Recommend using SigDescribe instead of KubeDescribe

https://github.com/kubernetes/kubernetes/blob/master/test/e2e/scheduling/framework.go#L21